packages/manager/modules/pci/src/projects/project/storages/containers/containers.controller.js

```diff
@@ -20,6 +20,37 @@ export default class PciStoragesContainersController {
     this.hasHighPerformanceStorage = this.hasHighPerformanceStorage();
   }
 
+  onPublicToggle(container) {
+    this.loadingContainer = container.id;
+    return this.$http
+      .put(`/cloud/project/${this.projectId}/storage/${container.id}`, {
+        containerType: container.state ? 'private' : 'public',
+      })
```

Comment:
prefer to create or use the dedicate HTTP call if exist here:
packages/manager/modules/pci/src/projects/project/storages/containers/containers.service.js

Reply:
Taken care asper your suggestion.